lab-steven/README.md

```diff
@@ -0,0 +1,63 @@
+#Dual Resource MongoDB API
+###Steven Bateman's Lab 13 for 401 JS
+
+###Overview
+This is an app that will allow you to practice POST, PUT, GET, and DELETE requests using HTTPIE against a database created with MongoDB that tracks a single resource, in the form of students with a name and age field.
+
+###Installation
+####Dependencies
```

Comment:
You probably shouldn't list httpie as a hard dependency, since you don't really NEED it for this project to work.